worker/lib/index.ts

```diff
@@ -1,16 +1,21 @@
 import * as bodyParser from 'body-parser';
-import { ChildProcess } from 'child_process';
+import { ChildProcess, exec } from 'child_process';
 import { multiWrite } from 'etcher-sdk';
 import * as express from 'express';
 import * as http from 'http';
 import { getSdk } from 'balena-sdk';
 import config = require("config");
 import {
+	getRuntimeConfiguration,
 	resolveLocalTarget,
 } from './helpers';
 import { TestBotWorker } from './workers/testbot';
 import QemuWorker from './workers/qemu';
 import { Contract } from '../typings/worker';
+import * as peripherals from './peripherals/peripherals.json';
```

Comment:
Peripherals have to be specified now? Are we making this a compulsory feature?

Reply:
@bucknalla they don't -have- to be specified. But if you want the testbot/workers contract to show that the peripheral is attached to it, you'll have to add it to that json file. 

Just an idea - this isn't set in stone :)

Reply:
Otherwise, there would have to be some quite clever scanning for attached peripherals - which maybe is a better idea now I think about it. But whatever the scan finds has to be then converted into something constrained, so the tests can determine if they should run or not.